cmdutil/redispool/redispool.go

```diff
@@ -41,6 +41,10 @@ type Config struct {
 	// When set to zero, aliveness is not checked.
 	IdleProbeInterval time.Duration `env:"REDIS_IDLE_PROBE_INTERVAL,default=0s"`
 
+	// Wait when true and the pool is at the MaxActiveConns limit, calls to Get()
+	// wait for a connection to be returned to the pool before returning.
+	Wait bool `env:"REDIS_WAIT,default=false"`
```

Comment:
NOTE: This is the default behaviour when you don't explicitly set it.

Reply:
Is there a timeout on how long we'll wait?

(Which is to say: do we think this is a case of a temporary spike in load and things will calm down if we wait a bit, or do we think it's connections getting leaked such that waiting will just show up as errors/timeouts later on?)

Reply:
I would guess that if we use [pool.GetContext](https://github.com/gomodule/redigo/blob/master/redis/pool.go#L198) on the consumer side we'd be ok.

Reply:
logtails is using `pool.GetContext` already in the hot path, `/drains`. When I update https://github.com/heroku/go-logplex/pull/327 to use this instead of forcing it I can update the `/stream` (aka subscribers) to use `pool.GetContext` as well.